mne/viz/_3d.py

```diff
@@ -1075,19 +1054,17 @@ def plot_alignment(info, trans=None, subject=None, subjects_dir=None,
                   rh=(0.5,) * 3)
     colors.update(skull_colors)
     for key, surf in surfs.items():
-        # Make a solid surface
-        mesh = _create_mesh_surf(surf, fig)
         with warnings.catch_warnings(record=True):  # traits
-            surface = mlab.pipeline.surface(
-                mesh, color=colors[key], opacity=alphas[key], figure=fig)
-        if key != 'helmet':
-            surface.actor.property.backface_culling = True
+            if key != 'helmet':
+                renderer.add_surface(surf, colors[key], alphas[key], True)
```

Comment:
try to keep explicit parameter names. For example here True it's not clear what it means. If you name it it becomes clear.

Reply:
I'm working on it.